creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.js

```diff
@@ -43,48 +44,94 @@ const mergePrimaryAndSecondaryEnabled = config.get(
  * @param {Object} param.syncRequestParameters axios params to make the sync request. Shape: { baseURL, url, method, data }
  */
 module.exports = async function ({
-  logger,
   syncType,
   syncMode,
-  syncRequestParameters
+  syncRequestParameters,
+  logger = genericLogger
```

Comment:
why'd you add a default here? every job processor is wrapped in a function that makes sure it gets passed a logger that has fields to filter by jobId and queue